app/helpers/cases_helper.rb

```diff
@@ -274,8 +274,14 @@ def action_link_for_destroy_case_link(kase, linked_case)
 
   def request_details_html(kase)
     content_tag(:strong, "#{kase.subject} ", class: 'strong') +
-      if kase.type == "Case::SAR::Offender" && kase.third_party_name.present?
-        content_tag(:div, kase.third_party_name, class: 'case-name-detail')
+      if kase.type == "Case::SAR::Offender"
```

Comment:
Small thing but I think we may have a bit of redundant logic here as we are asking if a case is third party twice when we could just have:

```ruby
      if kase.type == "Case::SAR::Offender" && kase.third_party?
        if kase.third_party_company_name.present?
          content_tag(:div, kase.third_party_company_name, class: 'case-name-detail')
        elsif !kase.third_party_company_name.present?
          content_tag(:div, kase.third_party_name, class: 'case-name-detail')
        else
          content_tag(:div, kase.subject, class: 'case-name-detail')
        end
```

Reply:
Thanks Will, I've refactored with this logic.